jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/PerspectiveIds.java

```diff
@@ -23,4 +23,18 @@
     String SEARCH_PARAMETER_TASK_ID = "taskId";
     String SEARCH_PARAMETER_JOB_ID = "jobId";
     String SEARCH_PARAMETER_IS_ERROR_ACK = "isErrorAck";
-}
+
+    String PROCESS_INSTANCE_DETAILS_SCREEN = "ProcessInstanceDetailsScreen";
```

Comment:
@bsremac @nmirasch wdyt?

Reply:
@cristianonicolai Looks good. Clear and concise.

Reply:
@cristianonicolai good!